libevmasm/RuleList.h

```diff
@@ -233,6 +233,30 @@ std::vector<SimplificationRule<Pattern>> simplificationRuleListPart5(
 		});
 	}
 
+	// Replace SHL >=256, X with 0
+	rules.push_back({
+		{Instruction::SHL, {A, X}},
+		[=]() -> Pattern {
```

Comment:
Perhaps pull the two functions each into a singe line so it is easier to see the four components?